README.md

```diff
@@ -1,34 +1,51 @@
-___Tanenbaum___
+**_Tanenbaum_**
 
 ![Tanenbaum logo](https://github.com/Sixstring982/tanenbaum/blob/main/public/logo.webp?raw=true "Tanenbaum logo")
 
 # Summary
 
 _Tanenbaum_ is an OCaml starter project for Advent of Code.
 
-
 ## Feature overview
 
-* Automatic downloading + caching of puzzle inputs
-* CLI for running puzzles + submitting answers to `adventofcode.com`
-* Automatic puzzle detection + registration
+- Automatic downloading + caching of puzzle inputs
+- CLI for running puzzles + submitting answers to `adventofcode.com`
+- Automatic puzzle detection + registration
+- ReasonML support out of the box
 
 # Quick-start
 
 ## Project setup
 
 First, clone or fork this repo.
 
+### Create your opam switch
```

Comment:
**Nit:** Might want to mention that, if using nix, you can run `nix develop` and skip all of these other setup steps